rust/lon/src/cli.rs

```diff
@@ -452,11 +473,20 @@ fn bot_fallible(directory: impl AsRef<Path>, forge: &impl Forge, base_ref: &str)
             .update()
             .with_context(|| format!("Failed to update {name}"))?;
 
-        let Some(summary) = summary else {
+        let Some(mut summary) = summary else {
             log::info!("No updates available");
             continue;
         };
 
+        let mut body: Option<String> = None;
```

Comment:
We can avoid declaring this mutable by just using a let if else statement below.